### ISSUE TITLE
Bread crumb markup and use of H2

### ISSUE THREAD

**Issue by timo54** [ATLAS-Illinois/Illinois-Framework-Theme#1081]
Sites like Actuarial Science and Risk Management (https://asrm.illinois.edu) are using a Breadtrail component that presents some accessibility issues. Here is a screen capture of the problem area followed by a list of the issues.

<img width="1836" height="558" alt="Image" src="https://github.com/user-attachments/assets/1b488b58-974c-4642-980e-82e13bc23755" />

1. H-tags should not be hidden. They are meant to visually  convey the content layout of the page for a number of disabled audiences. 
2. The “home” designation for the H2 is not a good use of an H-tag. It removes the H2 from the hierarchical sequential flow and causes the order to go H1 > H3, which is not logical. 
3. The NAV aria-label is perfect for alerting screen reader users in identifying the Breadcrumb  area. Using the H2 to do the same is unnecessarily redundant. 
<del>4. Putting the breadcrumb items in an Ordered list is misuse of a List. This should be an Unordered List.</del>

**Comment by timo54** [comment]
Please disregard item number 4. After review I realized the Breadcrumb is an Ordered List so in this context OL is appropriate.

**Comment by trubach** [comment]
If I'm understanding this correctly, simply removing the H2 element from the template (and breadcrumb) will fix the issue. The issue only applies to the 4.x branch. 

https://github.com/web-illinois/illinois_framework_theme/blob/62585e4bddc03c476040c9ad587544e163a033fe/templates/navigation/breadcrumb.html.twig#L12
